Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -46,6 +50,21 @@ final class SitterProfileViewModel: ObservableObject {
             .save(data, for: KeyValueStorage.Key.currentSitter)
     }
 
+    /// Uploads the pet sitter's data to the backend server asynchronously.
+    ///
+    /// - Throws: An error of type `Error` if a network error occurs during the upload process.
```

Comment:
What is the `Error` type? Might you mean `NetworkError`?

Reply:
Implementation changed